src/utils/user_utils.js

```diff
@@ -33,20 +33,37 @@ export function displayUsername(user, teammateNameDisplay) {
     return name;
 }
 
+export function rolesIncludePermission(roles, permission) {
+    const rolesArray = roles.split(' ');
+    return rolesArray.indexOf(permission) !== -1;
```

Comment:
maybe you can do `return rolesArray.includes(permission)`  and that will return a boolean